TimeTrackerManager/TimeTrackerManager/Core/Controller/Clients/AddClientsViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AddProjectsViewController.swift
+//  TimeTrackerManager
+//
+//  Created by Bocanu Mihai on 25.01.2022.
+//
+
+import UIKit
+
+class AddClientsViewController: UIViewController {
+
+    @IBOutlet var clientName: UITextField!
+    @IBOutlet var clientProject: UITextField!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    let clientPublisher = FirebaseClientPublisher()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        activityIndicator.isHidden = true
+
+    }
+    @IBAction func addClientButtonPressed(_ sender: Any) {
+        clientPublisher.createClient(clientName.text ?? "", project: clientProject.text ?? "") { [weak self] result in
```

Comment:
aici in loc sa creezi un client cu nume gol mai bine faci guard si te asiguri ca ai text in field-uri